fetch/http-cache/cc-request.html

```diff
@@ -202,7 +202,7 @@
               ["Cache-Control", "only-if-cached"]
             ],
             expected_status: 504,
-            expected_response_text: ""
+            expected_response_text: null
```

Comment:
Why do we not want to verify the contents of the body?

Reply:
Since the 504 is generated by the cache, not by the origin, it can contain anything; it's not under control of the test suite.

(I'm also starting to use these test cases for intermediary cache testing, and some of them do put things in there)